prow/tests/jobs_test.go

```diff
@@ -46,33 +47,42 @@ func TestMain(m *testing.M) {
 }
 
 func TestTrustedJobs(t *testing.T) {
-	const trusted = "test-infra-trusted"
-	trustedPath := path.Join(*jobConfigPath, "GoogleCloudPlatform", "oss-test-infra", "gcp-oss-test-infra-config.yaml")
+	var protectedClusters = map[string][]string{
```

Comment:
`TestTrustedJobs` is to ensure the `trusted` context behaves correctly.

Please do not complicate the `TestTrustedJobs` logic.
Move any knative stuff into its own test method and/or file.

Reply:
Make sense, updated. PTAL